runtime/queries/column_timeseries.go

```diff
@@ -116,22 +114,25 @@ func (q *ColumnTimeseries) Resolve(ctx context.Context, rt *runtime.Runtime, ins
 		Query:    sql,
 		Priority: priority,
 	})
-	defer DropTempTable(olap, priority, temporaryTableName)
+	defer dropTempTable(olap, priority, temporaryTableName)
 	if err != nil {
 		return err
 	}
 	rows.Close()
+
 	rows, err = olap.Execute(ctx, &drivers.Statement{
 		Query:    `SELECT * from "` + temporaryTableName + `"`,
 		Priority: priority,
 	})
 	if err != nil {
 		return err
 	}
```

Comment:
Do these `rows` get closed?

Reply:
I see there's a defer close in `convertRowsToTimeSeriesValues` – let's move that statement up here to make it clear